src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/module/NightHoursAccrualModule.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.homeoffice.digital.sas.balancecalculator.module;
+
+import static uk.gov.homeoffice.digital.sas.balancecalculator.utils.RangeUtils.UK_TIME_ZONE;
+
+import com.google.common.collect.Range;
+import com.google.common.collect.RangeSet;
+import com.google.common.collect.TreeRangeSet;
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual.enums.AccrualType;
+
+public class NightHoursAccrualModule extends AccrualModule {
+
+  public NightHoursAccrualModule() {
+    super(AccrualType.NIGHT_HOURS);
+  }
+
+  @Override
+  public BigDecimal calculateShiftContribution(ZonedDateTime startTime, ZonedDateTime endTime) {
```

Comment:
I thought we checked for having javadocs on public methods. I was looking to more easily understand what the private method was doing which wouldn't be covered in this case but then realised that our javadocs have disappeared.

Reply:
Added a doc comment to the method

Reply:
More to my point was that this should probably not have been copied and pasted into this repo
https://github.com/UKHomeOffice/callisto-balance-calculator/blob/main/checkstyle_suppressions.xml

Reply:
Discussed with @joannafawl  and agreed it's a topic wider than just this PR so will resolve this conversation without further action